### PR TITLE
refactor: removal of `initMovexWithLocalMaster` function and related unused dependencies

### PR DESCRIPTION
**Pull request by Abhishek-Mallick**
Status:
Closes: https://github.com/movesthatmatter/movex/issues/59
- [x] Have safely removed `initMovexWithLocalMaster` function
- [x] Unused imports removed

### CODE REVIEW

File: libs/movex/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (GabrielCTroia) [inline libs/movex/package-lock.json:1]: This file got generated when you ran 'npm install' I believe. We use `yarn` for the whole project so we don't need this, as it'll create conflicts. Could you please remove it? 🙏

File: libs/movex/src/lib/client/init.ts [hunk 2]
```diff
@@ -62,114 +57,3 @@ export const initMovex = <TResourceMap extends BaseMovexDefinitionResourcesMap>(
 // const masterServer = new MovexMasterServer({
 //   [resourceType]: masterResource,
 // });
-
```
Comment (GabrielCTroia) [inline libs/movex/src/lib/client/init.ts:65]: You can safely remove the comments above as well, starting with line 54. 🙏

**Review by GabrielCTroia — CHANGES_REQUESTED** [review]
Just two small changes and it's good to go! Thank you

**Comment by Abhishek-Mallick** [comment]
Should now be good commit: f6dae88b589b4adb441cfe71f1afb783819a7b8e

**Review by GabrielCTroia — APPROVED** [review]

**Comment by GabrielCTroia** [comment]
One last thing:

Your first 2 commits are unsigned and I can't merge, could you please sign them as well! See this https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification or use this directly `git rebase --exec 'git commit --amend --no-edit -n -S' -i YOUR_BRANCH`

**Comment by GabrielCTroia** [comment]
After this your PR will be merged to main! 🙏🚀

**Comment by Abhishek-Mallick** [comment]
I having difficulties verifying the first two commits
The command dosent change anything
Please look through this @GabrielCTroia

**Comment by GabrielCTroia** [comment]
Hey @Abhishek-Mallick , just merged it! Sorry for the struggle and thank you so much for you effort!

One last thing, could you please star the repo so we can bump it up today for #hacktoberfest!? ❤️🚀🙏

And also, do you have any feedback for the project? 
- did you find it easy to navigate or to understand the docs

**Pull request merged**